src/cpu/x64/cpu_isa_traits.hpp

```diff
@@ -34,7 +34,9 @@
 //#define XBYAK_USE_MMAP_ALLOCATOR
 /* Use Xbyak's memfd-based allocation, if available */
 #define XBYAK_USE_MEMFD
+#ifndef DNNL_ENABLE_EXCEPTIONS
```

Comment:
Do we need the exception in Release build as well?

Reply:
Why not ?
OpenVINO has exceptions even in release mode, for example if some operation is not supported, or parameters of operation is not valid

Reply:
That is reasonable question. Exceptions handling is additional routine and may affect kernel compilation time. As a result the first inference time and dynamic shape cases can be affected. Please use it just for debugging.